documentation/Spring-native-vs-Quarkus.md

```diff
@@ -0,0 +1,69 @@
+# Spring native vs Quarkus
+
+Nowadays it is very common to write and application and deploy it to cloud. Serverless and FaaS has become very popular. To deploy a Java application in latest cloud environment there are lot of challenges but major problem faced by developers is, memory footprint and startup time required for Java application. With introduction of framework like micronaut and microprofile java processes are getting faster and lightweight. In similar context Spring has introduced **Spring-native Beta version** and also Red Hat has introduced a new framework called **Quarkus**. This document aims at discussing pros and cons of this two frameworks and which is more suitable with devonfw and why?
+
+## Quarkus
+
+Quarkus is "_A Kubernetes Native Java stack tailored for OpenJDK HotSpot and **GraalVM**, crafted from the best of breed Java libraries and standards_". It is designed around a **container first philosophy**. i.e it is is optimized for low memory usage and fast startup times. 
+
+Quarkus achieves it in following ways:
+
+* First Class Support for GraalVM :
+
+  Use of AOT compilation
+* Build Time Metadata Processing : 
+
+  As much processing as possible is done at build time, so your application will only contain the classes that are actually needed at runtime. This results in less memory usage, and also faster startup time as all metadata processing has already been done.
+* Reduction in Reflection Usage:
+
+  As much as possible Quarkus tries to avoid reflection, reducing startup time and memory usage.
+* Native Image Pre Boot:
+
+  When running in a native image Quarkus pre-boots as much of the framework as possible during the native image build process. This means that the resulting native image has already run most of the startup code and serialized the result into the executable, resulting in even faster startup.
+
+## Spring- native:
+
+Current version of Spring Native 0.9.1 is designed to be used with Spring Boot 2.4.4
```

Comment:
I think it's good to provide version names to relate your information to the versions.
Anyhow, you should never say "the current version is ****" as this information then needs constantly to be updated.
But you can easily say, as of the versions compared here (0.9.1) etc.
So you provide a reference for the implementation, but you don't put the guarantee on it, that this is the latest information